Makefile

```diff
@@ -61,7 +61,14 @@ $(DOCKER_DISTRIB):
 	curl -o $(DOCKER_DISTRIB) $(DOCKER_DISTRIB_URL)
 
 tests:
-	go test -cover -tags netgo ./...
+	echo "mode: count" > profile.cov
+	for dir in $$(find . -type f -name '*_test.go' | xargs -n1 dirname | sort | uniq); do \
+	    go test -tags netgo -covermode=count -coverprofile=$$dir/profile.tmp $$dir;       \
+	    if [ -f $$dir/profile.tmp ]; then                                                 \
+	        cat $$dir/profile.tmp | tail -n +2 >> profile.cov;                            \
+	        rm $$dir/profile.tmp;                                                         \
+	    fi                                                                                \
+	done
```

Comment:
Shouldn't profile.cov be added to .gitignore? Also, what happens to this file? I.e. when I run this manually, how do I look at the results?